code/game/objects/items/ego_weapons/non_abnormality/carnival.dm

```diff
@@ -0,0 +1,28 @@
+
+/obj/item/ego_weapon/city/carnival_spear
+	name = "Carnival Spear"
+	desc = "A spear that the Carnival uses to hunt down their prey."
+	icon_state = "carnival_spear"
+	special = "If the target hit is a dawn foe, Deal triple damage to them."
+	force = 44
+	reach = 2
+	attack_speed = 1.2
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("whacks", "slashes")
+	attack_verb_simple = list("whack", "slash")
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 60,
+							TEMPERANCE_ATTRIBUTE = 60,
+							JUSTICE_ATTRIBUTE = 60
+							)
+
+/obj/item/ego_weapon/city/carnival_spear/attack(mob/living/target, mob/living/user)
+	if(target.stat == DEAD)
+		return
+	var/initial_force = force
+	if(istype(target, /mob/living/simple_animal/hostile/ordeal/amber_bug) || istype(target, /mob/living/simple_animal/hostile/ordeal/green_bot) || istype(target, /mob/living/simple_animal/hostile/ordeal/indigo_dawn) || istype(target, /mob/living/simple_animal/hostile/ordeal/steel_dawn))
```

Comment:
This should be moved to a list variable in the weapon.
I recommend something like 
```
var/list/empowered_targets = typecacheof(list(
	/mob/living/simple_animal/hostile/ordeal/amber_bug, 
	/mob/living/simple_animal/hostile/ordeal/green_bot,
	/mob/living/simple_animal/hostile/ordeal/indigo_dawn, 
	/mob/living/simple_animal/hostile/ordeal/steel_dawn
	))
```
That way this `if` statement can become `if(is_type_in_typecache(target, empowered_targets)`